testWaitingQueue(io.smallrye.faulttolerance.bulkhead.BulkheadTest) is flaky

```
[ERROR] Tests run: 1, Failures: 1, Errors: 0, Skipped: 0, Time elapsed: 5.873 s <<< FAILURE! - in io.smallrye.faulttolerance.bulkhead.BulkheadTest
[ERROR] testWaitingQueue(io.smallrye.faulttolerance.bulkhead.BulkheadTest)  Time elapsed: 1.991 s  <<< FAILURE!
java.lang.AssertionError
	at io.smallrye.faulttolerance.bulkhead.BulkheadTest.testWaitingQueue(BulkheadTest.java:62)
```

as seen on https://semaphoreci.com/smallrye/smallrye-fault-tolerance/branches/pull-request-137/builds/1

--- a/testsuite/basic/src/test/java/io/smallrye/faulttolerance/bulkhead/BulkheadTest.java
+++ b/testsuite/basic/src/test/java/io/smallrye/faulttolerance/bulkhead/BulkheadTest.java
@@ -59,7 +59,7 @@ public class BulkheadTest {
         for (int i = 0; i < loop; i++) {
             futures.add(pingService.ping(startLatch, endLatch));
         }
-        assertTrue(startLatch.await(500L, TimeUnit.MILLISECONDS));
+        assertTrue(startLatch.await(1000L, TimeUnit.MILLISECONDS));
         Thread.sleep(500L);
         // Next invocation should not make it due to BulkheadException
         try {

